templates/etc_systemd_system_postgresql.service.d_custom.conf.j2

```diff
@@ -0,0 +1,11 @@
+# {{ ansible_managed }}
+# Systemd unit file override to specify user/group as well as separate config
+# and data directories.
+[Service]
+User={{ postgresql_service_user }}
+Group={{ postgresql_service_group }}
+
+Environment=PGDATA={{ postgresql_conf_directory }}
+
+ExecStartPre=
+ExecStartPre=/usr/pgsql-9.3/bin/postgresql93-check-db-dir {{ postgresql_data_directory }}
```

Comment:
I can't say, I don't use CentOS!
I am a bit reluctant to add the CentOS support to this role, but if there is enough people to maintain and test it as it adds some complexity. That said, I am not the project leader, my word does not count, it's just my opinion ;)